extensions-core/lookups-cached-global/src/main/java/io/druid/query/lookup/NamespaceLookupExtractorFactory.java

```diff
@@ -95,7 +96,12 @@ public NamespaceLookupExtractorFactory(
   public boolean start()
```

Comment:
This method is gonna chomp interrupts, is that okay? Does the interrupt status matter for whatever thread "start" is called in?